### DISCUSSION TITLE
Gradle Action Stuck on QUEUED Forever

### DISCUSSION THREAD

**Discussion by simplemes-old** [community/community#26353] [Actions]
<p>I have an action to be triggered on push to the master branch.  It has been queued for 2+ hours now.  I tried to cancel the job, but it still remains.  I then tried another push and it is stuck at QUEUED too.</p>
<p>The project is in a private repo: <a href="https://github.com/simplemes/simplemes-core/actions" rel="nofollow noopener">https://github.com/simplemes/simplemes-core/actions</a></p>
<p>Any ideas on how to make this run?</p>
<p>The workflow is for a Composite Gradle build:</p>
<pre><code>name: Java CI

on: [push]

jobs:
build:

runs-on: ubuntu-latest

steps:
- uses: actions/checkout@v1
- name: Set up JDK 1.8
uses: actions/setup-java@v1
with:
java-version: 1.8
- name: Build with Gradle
run: ./gradlew assembleAll
</code></pre>

**Comment by dzivalli** [comment]
<p>same here, seems like it got broken. Didn’t find a status page for github actions</p>

**Comment by danmichaelo** [comment]
<p>Same here. Good to see I’m not the only one having this problem! Freaked out a little bit at first when the workflow couldn’t be cancelled and there was no status information.</p>

**Comment by StuartFerguson** [comment]
<p>I am having the same issue.  There doesnt seem to be anything on the Github Status page indicating that this is a known issue.</p>
<p>Is there anyone at Github on here that can comment on this issue?</p>

**Comment by drypa** [comment]
<p>Same to me</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/3a11bafc8eec6a4076e05bdc8870b105ae73588e.png" data-download-href="/uploads/short-url/8hHGbWMVt61HczLdz7PHmq845dQ.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/523874/181097295-a8f6d27d-1121-4fd1-88e7-8bc48caf079d.png" alt="image.png" data-base62-sha1="8hHGbWMVt61HczLdz7PHmq845dQ" width="690" height="137" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/3a11bafc8eec6a4076e05bdc8870b105ae73588e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">991×198 12.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>When I try to cancel those builds I get</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d3f7bd78c8853484f7f2e5980d493c6fc1e0fabc.png" data-download-href="/uploads/short-url/uf9wFUJdey4BclQGg00AfqvL9vu.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/523874/181097296-1c75cd1f-ac63-46e6-a71b-16c59f0cc56c.png" alt="image.png" data-base62-sha1="uf9wFUJdey4BclQGg00AfqvL9vu" width="690" height="173" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d3f7bd78c8853484f7f2e5980d493c6fc1e0fabc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">856×215 7.12 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>on <a href="https://www.githubstatus.com/" rel="nofollow noopener">https://www.githubstatus.com</a> no incidents reported days ago</p>
<p>did you resolve this situation?</p>

**Comment by bbwharris** [comment, marked as answer]
<p>You need to fix your indention, the default Gradle Build action is incorrect:</p>
<pre><code class="lang-auto">
    name: Java CI
    
    on: [push]
    
    jobs:
      build:
        runs-on: ubuntu-latest
        steps:
          - uses: actions/checkout@v1
          - name: Set up JDK 1.8
            uses: actions/setup-java@v1
            with:
              java-version: 1.8
          - name: Unit Tests
            run: ./gradlew :app:testDebugUnitTest

Is working for my case

</code></pre>

**Comment by simplemes-old** [comment]
<p>Thanks, this was the cause.  YML and indenting/blank lines again :).</p>

**Comment by andybirenzi** [comment]
<p>I’m having the same is issue. GIT actions flow is stuck in queue for a long time, more than 43 min. Is this a widespread is issue we should know about?</p>

**Comment by iamsteeve** [comment]
<p><a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a> Github actions degraded <a class="mention" href="/andybirenzi">@andybirenzi</a></p>
